### DISCUSSION TITLE
documento 15 maggio non funziona

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#528] [Richieste di aiuto] [answered]
Buongiorno ,volevo comunicare solo che il documento del 15 maggio risulta come se fosse bloccato ,in quanto il registro mi indica sempre il seguente avviso "Non è previsto il caricamento del documento del 15 maggio per questa cattedra".Come devo fare per le classi quinte ?

**Comment by trinko** [comment, marked as answer]
Il documento del quindici maggio può essere caricato solo dal docente coordinatore della classe quinta.
